bucket/potplayer.json

```diff
@@ -0,0 +1,71 @@
+{
+    "version": "1.7.20538",
+    "description": "Highly customizable media player",
+    "homepage": "https://potplayer.daum.net/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://potplayer.daum.net/publicRelation"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://t1.daumcdn.net/potplayer/PotPlayer/Version/20190919_1.7.20538/PotPlayerSetup64.exe#/cosi.7z",
+            "hash": "c54f447ccd217cb07aca6199722054dd0e2ae97b07aa946192cd4e21b3db7b68",
+            "shortcuts": [
+                [
+                    "PotPlayer64.exe",
+                    "PotPlayer"
+                ],
+                [
+                    "PotPlayerMini64.exe",
+                    "PotPlayer Mini"
+                ]
+            ]
+        },
+        "32bit": {
+            "url": "https://t1.daumcdn.net/potplayer/PotPlayer/Version/20190919_1.7.20538/PotPlayerSetup.exe#/cosi.7z",
+            "hash": "c7e7ee44e45cb7d09eac7a25c7710cb619eb6759df6a05439aa2d584233b9770",
+            "shortcuts": [
+                [
+                    "PotPlayer.exe",
+                    "PotPlayer"
+                ],
+                [
+                    "PotPlayerMini.exe",
+                    "PotPlayer Mini"
+                ]
+            ]
+        }
+    },
+    "installer": {
```

Comment:
Convert to pre_install

Reply:
What's the reasoning on when to choose `pre_install` and when `installer`?

Reply:
More egonomic / shorter manifest. Also meaning. This is not installer process. Are you installing anythig?

Reply:
Hard to tell sometimes. Is creating config files "installing"? I just used what you had.

Reply:
Yes in my case it is critical part of installation. But when you are removing only some folders, then consider it as installer is not appropriate.

Main argument is ergonomic. Because it is better to have 1 line instead of 4

Reply:
Well, since I had to create .ini files to disable autoupdate, that part is now pretty much equivalent to your manifest.